www/viewgame

```diff
@@ -1676,6 +1670,7 @@ if (!isEmpty($genre)) {
         }
 
         // check for explicit cross-recommendations
+        $sandbox = $mysandbox ? "1" : "and not(u.sandbox) and not (u2.sandbox)";
```

Comment:
The "else" string starts with `and`, but the SQL below already does `and $sandbox`.

Doesn't that expand to `and and`?

Reply:
Good catch! Fixed.